pkg/core/interop/contract/call.go

```diff
@@ -69,6 +69,25 @@ func Call(ic *interop.Context) error {
 		return fmt.Errorf("method not found: %s/%d", method, len(args))
 	}
 	hasReturn := md.ReturnType != smartcontract.VoidType
+
+	if ic.SaveInvocations {
+		var (
+			arrCount  = len(args)
+			truncated = false
+			argBytes  []byte
+		)
+		if argBytes, err = ic.DAO.GetItemCtx().Serialize(stackitem.NewArray(args), false); err != nil {
+			truncated = true
+		}
+
+		ic.InvocationCalls = append(ic.InvocationCalls, state.ContractInvocation{
```

Comment:
We're missing the code responsible for reverted execution. We should either revert invocations along with context unloading if exception has been thrown (exactly the same way as it's done for `ic.Notifications`) or at least we need to set a proper status (ref. https://github.com/nspcc-dev/neo-go/pull/3569#discussion_r1746550305).

Reply:
Discussed with Roman, let's now keep more simple version that does not revert invocation records. So just keep every record even if transaction is FAULTed or if execution is reverted due to exception. This behaviour diverges from what we have for Notifications, but it's OK for the initial prototype. This behaviour also needs to be properly documented at the `./docs/rpc.md`.